mdata/aggmetric_test.go

```diff
@@ -486,14 +486,13 @@ func TestGetAggregated(t *testing.T) {
 	m.Add(30, 30)
 	m.Add(31, 31)
 	m.Add(32, 32)
-	m.Add(40, 40)
```

Comment:
why is this removed?

Reply:
I removed it to test the bug fix. The span is 5 and 40 is the boundary. If (40, 40) is there it behaves the same (aggregator will flush and `len(futurePoints)==0`).
I can also add extra points like m.Add(41, 41) m.Add(42, 42) to test the 45 boundary but I think they are kind of the same. I can update this if you prefer.

Reply:
I removed it to test the bug fix. The span is 5 and 40 is the boundary. If (40, 40) is there it behaves the same (aggregator will flush to the underlying aggMetric).
So if (40, 40) is removed, before this PR, ({31+32, Ts:35}) will not be in the result.
I can also add extra points like `m.Add(41, 41)` `m.Add(42, 42)` to test the 45 boundary but I think they are kind of the same. I can update this if you prefer.

Reply:
makes sense, what you have.